lnrpc/rpc.proto

```diff
@@ -859,7 +862,11 @@ message SendManyRequest {
 
     // A manual fee rate set in sat/vbyte that should be used when crafting the
     // transaction.
-    int64 sat_per_byte = 5;
+    uint64 sat_per_vbyte = 4;
```

Comment:
Strange (and convenient) that field 4 is missing, but looks like it was just an onversight in `9b1f21e2`.